ui/apps/platform/src/services/AuthService/AuthService.ts

```diff
@@ -147,7 +148,7 @@ export function updateAuthProvider(authProvider: AuthProvider): Promise<AuthProv
  * Saves auth provider either by creating a new one (in case ID is missed) or by updating existing one by ID.
  */
 export function saveAuthProvider(authProvider: AuthProvider): string | Promise<AuthProvider> {
-    if (authProvider.active) {
+    if (authProvider.active || authProvider.unmodifiable) {
         return authProvider.id;
     }
     return authProvider.id
```

Comment:
Not yours, but you touched it. Why this condition?

Reply:
`saveAuthProvider` is called when you press `Save` in the auth provider form UI. This button can be pressed when auth provider config is unchanged, but groups change. 

-->This change prevents this code from returning an error when someone changes the groups for auth provider.

Reply:
Ah, I didn't expand the code below, sorry.
Not yours, but that's a bad practice to leave something on the next line after return.

Reply:
As Michaël said, not yours, but we are reviewing it :)

The only 2 calls of the `saveAuthProvider` service function are in authSagas.js file above.

@ivan-degtiarenko To avoid inconsistency at the risk of temporary code duplication, what do you think about:

1. Add comment in `saveAuthProvider` function that `if` statement might be deleted it the future, because redundant with condition in authSagas.js file. It seems like condition at point of call not to make a request is clearer than invisible early return in the service function.

2. Include both conditions in authSagas.js file:

```diff
-            if (!remaining.unmodifiable) {
+            if (!remaining.active && !remaining.unmodifiable) {
                 yield call(AuthService.saveAuthProvider, remaining);
             }
```